DOC: Add HGBT on landing page

GaelVaroquaux opened the pull request:
Replace SVR/SVM by a pointer to the HGBT

GaelVaroquaux committed:
DOC: Add HGBT on landing page

And remove SVR/SVM

ogrisel reviewed (approved):
Shall we also make Ridge and Logistic Regression (or "Linear models") more preeminent?

GaelVaroquaux committed:
Also change the clustering:

Point to HDBSCAN and hierarchical clustering rather than spectral
clustering and mean shift

GaelVaroquaux committed:
Clean the "whats_new" on the landing page

GaelVaroquaux commented:
By popular demand, also changed the clustering algorithms that we point to and remove extra whats new.

GaelVaroquaux committed:
Update maintainer instructions

Suggest to remove old whats_new entries on the landing page

GaelVaroquaux committed:
Add ridge and logistic regression

GaelVaroquaux commented:
@ogrisel : I added ridge and logisitc
